test/instrumentation/update-documentation.sh

```diff
@@ -24,6 +24,12 @@ set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
 source "${KUBE_ROOT}/test/instrumentation/stability-utils.sh"
+source "${KUBE_ROOT}/hack/lib/version.sh"
 
+# extract version env variables so we can pass them in
+kube::version::get_version_vars
+
+# update the documented list of metrics
 kube::update::documentation::list
-kube::update::documentation
+# now write the actual documentation file
+kube::update::documentation $KUBE_GIT_MAJOR $KUBE_GIT_MINOR
```

Comment:
```suggestion
kube::update::documentation "$KUBE_GIT_MAJOR" $KUBE_GIT_MINOR
```

```
In ./test/instrumentation/update-documentation.sh line 35:
kube::update::documentation $KUBE_GIT_MAJOR $KUBE_GIT_MINOR
                            ^-------------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
kube::update::documentation "$KUBE_GIT_MAJOR" $KUBE_GIT_MINOR

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```